cmd/solana_exporter/exporter.go

```diff
@@ -103,13 +107,31 @@ func NewSolanaCollector(
 		isHealthy: prometheus.NewDesc(
 			"solana_is_healthy",
 			"Whether the node is healthy or not.",
-			nil,
+			[]string{IdentityLabel},
```

Comment:
I am not opposed to adding the `identity` label here (and to the other new metrics), as it would different the results from multiple exporters running on multiple nodes. 

However, just thinking out loud here: There are other metrics such as `solana_confirmed_transactions_total` and `solana_confirmed_slot_height` which are also in theory specific to the node being queried. In fact, all the metrics are. Should we not then add the `identity` label to ALL metrics? If not all, then how do we decide which ones to add it to.

Thoughts? @SEJeff